src/main/java/org/jabref/gui/push/PushToTexShop.java

```diff
@@ -65,13 +65,17 @@ public void pushEntries(BibDatabaseContext database, List<BibEntry> entries, Str
     protected String[] getCommandLine(String keyString) {
         String citeCommand = getCitePrefix();
         // we need to escape the extra slashses
-        if (getCitePrefix().contains("\\")) {
-            citeCommand = "\"\\" + getCitePrefix();
+        int intSlashPosition = getCitePrefix().indexOf("\\");
+
+        if (intSlashPosition != -1) {
+            StringBuilder sb = new StringBuilder(getCitePrefix());
+            sb.insert(intSlashPosition, "\\");
+            citeCommand = sb.toString();
         }
 
         String osascriptTexShop = "osascript -e 'tell application \"TeXShop\"\n" +
                 "activate\n" +
-                "set TheString to " + citeCommand + keyString + getCiteSuffix() + "\"\n" +
+                "set TheString to \"" + citeCommand + keyString + getCiteSuffix() + "\"\n" +
```

Comment:
It seems that nested "" won't work. Think, we don't need to handle that cases?

Do we need some test cases here? 😅